src/common/logo.c

```diff
@@ -249,6 +269,37 @@ static void loadGarudaLogo(FFlogo* logo, bool doColor)
     sprintf(logo->chars[19], FASTFETCH_TEXT_MODIFIER_BOLT"%s                                              "FASTFETCH_TEXT_MODIFIER_RESET, color);
 }
 
+static void loadLogoFromFile(FFlogo* logo, bool doColor, char* pathToFile)
+{
+    strcpy(logo->name, "amogus");
```

Comment:
No need to do this, as we never load the logo by name. Maybe just set a null byte at index 0 to be save.